README.md

```diff
@@ -0,0 +1,53 @@
+# PowSyBl Tutorials
+
+PowSyBl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis. Its modular approach allows developers to extend or customize its features.
+
+PowSyBl is part of the LF Energy Foundation, a project of The Linux Foundation that supports open source innovation projects within the energy and electricity sectors.
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/powsybl/powsybl-gse/master/gse-spi/src/main/resources/images/logo_lfe_powsybl.svg?sanitize=true" alt="PowSyBl Logo" width="50%"/>
+</p>
+
+Read more at https://www.powsybl.org !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl.ddl@rte-france.com](mailto:powsybl.ddl@rte-france.com).
+
+
+## Tutorials
+
+This document describes how to build and run small tutorial projects. For more in depth explanations on each tutorial, please visit the https://www.powsybl.org/docs/tutorials page.
+
+## Environment requirements
+
+  * JDK *(1.8 or greater)*
+  * Maven *(3.3.9 or greater)*
+
+Most tutorials show simple code and can be run directly from maven using the exec:java goal:
```

Comment:
Make it explicit that this command must be done from the repository root (add ` cd powsybl-tutorials` before ?)